### PR TITLE
docs: add quotas

### PR DESCRIPTION
**Pull request by stefanicjuraj**
Fixes https://github.com/daytonaio/enterprise-docs/issues/150

### CODE REVIEW

**Comment by ivan-burazin** [comment]
@stefanicjuraj ?

**Review by ivan-burazin — APPROVED** [review]

File: src/content/docs/configuration/teams.md [hunk 1]
```diff
@@ -87,13 +85,13 @@ Each row contains the following information:
...
 
-1. In the dashboard sidebar, click the **`Settings`** page under the **Team** section.
+1. Navigate to the **`Settings`** page under the **Team** section.
 2. Enter your new team name in the input field provided under the **Team name** heading.
 3. Click the **`Save`** button.
 
 ## Delete a Team
 
-You can delete a team from your Daytona instance. This allows you to free up resources used under your Daytona subscription, or to recreate your team from scratch.
+You can delete a team from your Daytona instance. This allows you to free up resources used under your Daytona quota, or to recreate your team from scratch.
```
Comment (fabjanvucina) [inline src/content/docs/configuration/teams.md:94]: > used under your Daytona quota

you can delete this part of the sentence

File: src/content/docs/usage/usage.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
+---
+title: Usage
+description: Learn about the Daytona usage and how to manage it.
+---
+
+Daytona provides an overview of your resources usage within your current quota limits. This allows you to better understand where optimizations can be made to keep your usage under your current quota limits.
+
+Metadata in the usage page specifies the current state of resources and quota limits. If the usage does not exist for your team, the page will display *No usage found*.
```
Comment (fabjanvucina) [inline src/content/docs/usage/usage.mdx:8]: This paragraph is unnecessary

File: src/content/docs/usage/usage.mdx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  Example: `500.00/10020`
+
+- **Active Workspaces**
+
+  Specifies how many Workspaces are currently running versus the maximum number you can have active simultaneously.
+
+  Example: `0/3`
+
+- **Filter by Workspace ID**
```
Comment (fabjanvucina) [inline src/content/docs/usage/usage.mdx:24]: This should be extracted from the resources list. There should be paragraph that basically explains how to browse your workspace usage sessions.

image

File: src/content/docs/administration/quotas.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+1. Click the **`New +`** button to create a Quota.
+2. Enter the details for the Quota.
+
+- **Name**
+
+    The name of the Quota.
+
+    Example: `Basic`
+
+- **Workspace Credits**
```
Comment (fabjanvucina) [inline src/content/docs/administration/quotas.mdx:21]: If left unspecified, workspace usage will not be limited within the quota. (phrase it however you think is best)

File: src/content/docs/administration/quotas.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    Example: `3000`
+
+- **Workspace Idle Timeout**
+
+    Optionally, define the idle timeout period for the Workspace. If specified, Workspaces that have remained inactive longer than the idle timeout will be automatically stopped.
+
+    Example: `1h 30min`
+
+- **Parallel Workspaces**
+
```
Comment (fabjanvucina) [inline src/content/docs/administration/quotas.mdx:34]: Same as workspace credits -> unspecified = unlimited

File: src/content/docs/administration/quotas.mdx [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+    Optionally, specify the maximum number of Workspaces that can be active simultaneously.
+
+    Example: `5`
+
+- **Pinned Workspaces**
+
+    Optionally, specify the maximum number of Workspaces that can be pinned by the user.
```
Comment (fabjanvucina) [inline src/content/docs/administration/quotas.mdx:41]: Same as workspace credits -> unspecified = unlimited

File: src/content/docs/administration/quotas.mdx [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+3. Click **`Save`** to apply the changes.
+
+## Remove a Quota
+
+Daytona provides the option to remove a Quota if it is no longer required.
+
+1. Click **`Delete`** in the Quota card you want to remove.
+
+:::danger
+Removing the Quota will prevent the creation of new Workspaces within that Quota. Existing Workspaces will not be affected.
```
Comment (fabjanvucina) [inline src/content/docs/administration/quotas.mdx:62]: Removing the quota will affect the teams to which it was applied. These teams will lose access to resources until a different quota is applied to them.

**Review by fabjanvucina — CHANGES_REQUESTED** [review]

File: src/content/docs/usage/usage.mdx [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+  Example: `500.00/10020`
+
+- **Active Workspaces**
+
+  Specifies how many Workspaces are currently running versus the maximum number you can have active simultaneously.
+
+  Example: `0/3`
+
+View the resources usage within your current quota limits by accessing the **`Usage`** page in the dashboard sidebar. You will be presented with metadata specifying the current state of resources, used credits, active Workspaces list and limits. If the usage does not exist for your team, the page will display *No usage found*.
```
Comment (fabjanvucina) [inline src/content/docs/usage/usage.mdx:22]: _View the resources usage within your current quota limits by accessing the **`Usage`** page in the dashboard sidebar. Additionally, you can view the history of your Workspace usage sessions._

File: src/content/docs/administration/quotas.mdx (hunk 4 above)
Comment (fabjanvucina) [reply src/content/docs/administration/quotas.mdx:21]: If left empty, the Workspace _credits_ will not be limited by the Quota.

File: src/content/docs/administration/quotas.mdx (hunk 5 above)
Comment (fabjanvucina) [reply src/content/docs/administration/quotas.mdx:34]: If left empty, the _number of_ parallel Workspaces will not be limited by the Quota.

File: src/content/docs/administration/quotas.mdx [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+- **Parallel Workspaces**
+
+    Optionally, specify the maximum number of Workspaces that can be active simultaneously. If left empty, the parallel Workspaces will not be limited by the Quota.
+
+    Example: `5`
+
+- **Pinned Workspaces**
+
+    Optionally, specify the maximum number of Workspaces that can be pinned by the user. If left empty, the pinned Workspaces will not be limited by the Quota.
```
Comment (fabjanvucina) [inline src/content/docs/administration/quotas.mdx:41]: If left empty, the _number of_ pinned Workspaces will not be limited by the Quota.

**Review by fabjanvucina — CHANGES_REQUESTED** [review]

**Review by fabjanvucina — APPROVED** [review]

**Pull request merged**